### PR TITLE
Update jsp files from identity endpoints

### PR DESCRIPTION
**Pull request by RusJaI**

### CODE REVIEW

File: apps/recovery-portal/src/main/webapp/self-registration-username-request.jsp [hunk 1]
```diff
@@ -207,6 +266,9 @@
             }
         });
 
+        function onCompleted() {
+            $('#register').submit();
+        }
         function goBack() {
```
Comment (brionmario) [inline apps/recovery-portal/src/main/webapp/self-registration-username-request.jsp:272]:
```suggestion

        function goBack() {
```

File: apps/recovery-portal/src/main/webapp/self-registration-username-request.jsp [hunk 2]
```diff
@@ -68,13 +75,40 @@
     } else if (errorMsgObj != null) {
         errorMsg = errorMsgObj.toString();
     }
+
+    ReCaptchaApi reCaptchaApi = new ReCaptchaApi();
+        try {
```
Comment (brionmario) [inline apps/recovery-portal/src/main/webapp/self-registration-username-request.jsp:80]: Fix indentation here.

File: apps/recovery-portal/src/main/webapp/self-registration-username-request.jsp (hunk 1 above)
Comment (RusJaI) [reply apps/recovery-portal/src/main/webapp/self-registration-username-request.jsp:273]: fixed

File: apps/recovery-portal/src/main/webapp/self-registration-username-request.jsp (hunk 2 above)
Comment (RusJaI) [reply apps/recovery-portal/src/main/webapp/self-registration-username-request.jsp]: fixed

**Review by tgtshanika — APPROVED** [review]

**Review by HiranyaKavishani — APPROVED** [review]

**Pull request merged**